python/pyspark/sql/window.py

```diff
@@ -97,6 +97,33 @@ def rowsBetween(start, end):
         and ``Window.currentRow`` to specify special boundary values, rather than using integral
         values directly.
 
+        A row based boundary is based on the position of the row within the partition.
+        An offset indicates the number of rows above or below the current row, the frame for the
+        current row starts or ends. For instance, given a row based sliding frame with a lower bound
+        offset of -1 and a upper bound offset of +2. The frame for row with index 5 would range from
+        index 4 to index 6.
+        """
+        # from pyspark.sql import Window
```

Comment:
@jagadesh-kiran why is it commented? It should be started with `>>>` to make it a proper [doctest](https://docs.python.org/2/library/doctest.html).

Reply:
Thanks @HyukjinKwon  for your review, Using >>> compilation failure will happen for py file , so i will uncomment the same ,should it be fine ?